lib/service/Service.ts

```diff
@@ -0,0 +1,86 @@
+import assert from 'assert';
+import { GrpcComponents } from '../grpc/GrpcServer';
+import Logger from '../Logger';
+import Pool from '../p2p/Pool';
+import OrderBook from '../orderbook/OrderBook';
+import { orders, matchingEngine } from '../types';
+import LndClient from '../lndclient/LndClient';
+import RaidenClient, { TokenSwapPayload } from '../raidenclient/RaidenClient';
+import grpc from 'grpc';
+import { PairInstance } from '../types/db';
+import { GetInfoResponse } from '../proto/lndrpc_pb';
+import { Orders } from '../orderbook/OrderBookRepository';
+import { OwnOrder } from '../types/orders';
+
+export type XudInfo = {
+  lnd: GetInfoResponse,
+};
+
+/** Class containing the available RPC methods for XUD */
+class GrpcService {
+  private orderBook: OrderBook;
+  private lndClient: LndClient;
+  private raidenClient: RaidenClient;
+  private pool: Pool;
+  private shutdown: Function;
+  private logger: Logger;
+
+  /** Create an instance of available RPC methods and bind all exposed functions. */
+  constructor(components: GrpcComponents) {
```

Comment:
same as above. in addition, `GrpcComponents` should be removed from `GrpcServer.ts`

Reply:
Yeah this makes sense, maybe rename this type `ServiceComponents`?

Reply:
Maybe just define it as an anonymous type, since it's not really going to be used anywhere else.